docs/README.md

```diff
@@ -140,9 +140,9 @@ private_key_filename: my-private-key
 
 ### Setup your customer access control list (ACL)
 
-Edit `customers_url` in gateway_config.yml.
+Edit `customers_url` in gateway_config.yml?
 
-Enter a list of GitHub usernames for your customers, these are case-sensitive.
```

Comment:
Duplicate ? mentioned in more detail in `Before you begin` https://github.com/openfaas/openfaas-cloud/pull/357/files#diff-e70100059cc3a7376740bbe9751a082eL34

Suggest remove or mention the location of the file